setup.py

```diff
@@ -83,21 +82,19 @@ def read(f):
     return (here / f).read_text('utf-8').strip()
 
 
-class PyTest(TestCommand):
-    user_options = []
+NEEDS_PYTEST = {'pytest', 'test'}.intersection(sys.argv)
+pytest_runner = ['pytest-runner'] if NEEDS_PYTEST else []
 
-    def run(self):
-        import subprocess
-        errno = subprocess.call([sys.executable, '-m', 'pytest', 'tests'])
-        raise SystemExit(errno)
-
-
-tests_require = install_requires + ['pytest', 'gunicorn',
-                                    'pytest-timeout', 'async-generator']
+tests_require = ['pytest', 'gunicorn',
+                 'pytest-timeout', 'async-generator']
 
+name = 'aiohttp'
+appveyor_slug = 'asvetlov/{}'.format(name)  # FIXME: move under aio-libs/* slug
```

Comment:
@asvetlov how about finally fixing AppVeyor URL?

Reply:
I don't remember how to setup it.
Could you do it yourself?

Reply:
This requires owner privileges AFAIR, so I can't. It's a bit tricky, but the bottom line is that you create a team there mapped to the team in GitHub and give it some access there. Also you need to somehow create an org link there.

Reply:
Here you go: https://www.appveyor.com/docs/team-setup/#setting-up-appveyor-account-for-github-organization
Create a separate account called `aio-libs` and then add your personal account as its administrator. You can also create team mapped to some team in GitHub, allowing its members to cancel/rerun builds.